handbook/product/README.md

```diff
@@ -253,21 +253,28 @@ There are two types of tooltips. The two types of tooltips have some unique styl
 
 ## Release 
 
-This section outlines the communication between the product team, growth team, product team,
+This section outlines the communication between the product team, marketing team, product team,
 and customer success team prior to a release of Fleet.
 
 ### Goal
 
 Keep the business up to date with improvements and changes to the Fleet product so that all stakeholders can communicate
 with customers and users.
 
+### Product Marketing Tiers 
+At the kickoff of a product sprint, the marketing and product teams should decide which features are most important to highlight. When a feature gets scheduled for release the Tier helps marketing give it the biggest impact. The teams should prioritize the features that are posed to add the most value to our customers and/or unlock new revenue streams for the company. Once the features have been discussed the next step is to bucket them into tiers for marketing prioritization. The current tiers are 1-3. 
+
+- Tier 1: A separate product feature announcement. The most important release types, these require a specific and custom marketing package. Usually including an individual blog post, a demo video and potentially a press release or official product marketing launch. Due to limited availability there is only room for one Tier 1 product announcement per release sprint.
+- Tier 2: A highlighted feature in the release notes. This product feature will be highlighted at the top of the Sprint Release blog post. Depending on the feature specifics this will include: a 1-2 paragraph write-up of the feature, a demo video (if applicable) and a link to the docs. Ideally there would be no more than 3 *Tier 2* features in a release post, otherwise the top features will be crowded.
+- Tier 3: A feature worth mentioning in the changelog. In most cases a product feature will fit into this Tier. This includes 1-2 sentences in the Changelog and release blog post. 
+
 ### Blog post
 
-The product team is responsible for providing the [growth team](../growth/README.md) with the necessary information for writing
+The product team is responsible for providing the [marketing team](../marketing/README.md) with the necessary information for writing
 the release blog post. This is accomplished by filing a release blog post issue and adding
 the issue to the growth board on GitHub.
 
-The release blog post issue includes a list of the primary features included in the upcoming
+The release blog post issue includes a list of the primary *Tier 1* features included in the upcoming
```

Comment:
Do you mean Tier 2 and Tier 3 hear? The section above implies that Tier 1 features aren't included in release blog post (they get their own post) here: https://github.com/fleetdm/fleet/pull/10128/files#diff-5f7714f6b120efb9253d8ce52b7fe87629fe2b4f5d515a900c4f68dfe6336d34R267

I could be wrong.

Reply:
Correct! thanks for catching Noah.